packages/tenant-process/src/services/tenantService.ts

```diff
@@ -424,6 +431,90 @@ export function tenantServiceBuilder(
       return updatedTenant;
     },
 
+    async verifyVerifiedAttribute(
+      {
+        tenantId,
+        tenantAttributeSeed,
+        organizationId,
+        correlationId,
+      }: {
+        tenantId: TenantId;
+        tenantAttributeSeed: tenantApi.VerifiedTenantAttributeSeed;
+        organizationId: TenantId;
+        correlationId: string;
+      },
+      logger: Logger
+    ): Promise<Tenant> {
+      logger.info(
+        `Verifying attribute ${tenantAttributeSeed.id} to tenant ${tenantId}`
+      );
+
+      if (organizationId === tenantId) {
+        throw verifiedAttributeSelfVerificationNotAllowed();
+      }
+
+      const allowedStatuses = [
+        agreementState.pending,
+        agreementState.active,
+        agreementState.suspended,
+      ];
+      await assertVerifiedAttributeOperationAllowed({
+        producerId: organizationId,
+        consumerId: tenantId,
+        attributeId: unsafeBrandId(tenantAttributeSeed.id),
+        agreementStates: allowedStatuses,
+        readModelService,
+        error: attributeVerificationNotAllowed(
```

Comment:
The error is in input because this function is used also in revokeVerifiedAttribute (https://github.com/pagopa/interop-be-monorepo/pull/841) but with different error